tidb-binlog/proto/go-binlog/secondary_binlog.pb.go

```diff
@@ -2,21 +2,23 @@
 // source: secondary_binlog.proto
 
 /*
-	Package slave_binlog is a generated protocol buffer package.
-
-	It is generated from these files:
-		secondary_binlog.proto
-
-	It has these top-level messages:
-		Column
-		ColumnInfo
-		Row
-		Table
-		Key
-		TableMutation
-		DMLData
-		DDLData
-		Binlog
+Package slave_binlog is a generated protocol buffer package.
```

Comment:
Why this file here, do we still using it?

Reply:
It is hard to say. Let's keep it as it is.